.github/workflows/ci.yaml

```
@@ -11,7 +11,7 @@ jobs:
     steps:
       - name: Set current date as env variable
         id: tagname
-        run: echo "::set-output name=timestamp::$(date +'%s')"
+        run: echo "timestamp=$(date +'%s')" >> GITHUB_OUTPUT
```

Correct me if I'm wrong but based on the documentation I've found here: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

this should rather be:

```suggestion
        run: echo "timestamp=$(date +'%s')" >> "$GITHUB_OUTPUT"
```